packages/falter-berlin-tunnelmanager/files/tunnelman.sh

```diff
@@ -192,22 +186,28 @@ manage() {
     # Check for stale tunnels and tear em down
     while true; do
         for interface in $interfaces; do
-	    if [ $(get_age "$interface") -ge $OPT_TUNNEL_TIMEOUT ]; then
+            if [ $(get_age "$interface") -ge $OPT_TUNNEL_TIMEOUT ]; then
                 log "Tunnel to $interface timed out."
                 teardown "$interface"
-	    fi
-	done
+            fi
+        done
 
-	if [ $(echo $connections | wc -w) -lt $connection_count ]; then
-            ep=$(get_least_used_tunnelserver "$tunnel_endpoints")
-	    if [ ! -z "$ep" ]; then
+        tmp_tunnel_endpoints="$tunnel_endpoints"
+
+        while [ $(echo $connections | wc -w) -lt $connection_count ]; do
```

Comment:
how? Can you add the whole line?

Reply:
NVM, I confused something: In General `$#` can give you the number of arguments a function/script got called with. A second option is to use it like here https://github.com/freifunk-berlin/falter-packages/blob/56af937cded186ced5dd9d84078b30ec7794af8a/luci/luci-app-falter-owm/files/owm.sh#L310

Unfortunately, this can't get the number of space-seperated string, but the string-length only. ^^